### DISCUSSION TITLE
Correct copyright notice for Alire?

### DISCUSSION THREAD

**Discussion by m7a** [alire-project/alire#965] [Q&A] [answered]
Hello community,

I am trying to build a `.deb` package for `alr`. I understand Alire is licensed under GPLv3, but I have not found the typical "Alire ... Copyright (c) ..." line -- is there a copyright line available for Alire somewhere that I can use to get the attribution and license right?

Thanks in advance
Ma_Sys.ma

**Comment by mosteo** [comment]
Hi @m7a , can you point me to an example in another project of what you are looking for? TIA.

**Reply by m7a** [reply]
Hello,

thank you very much for the quick response.

Section 5.3.1 of <https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/> has some examples for the `Copyright` lines.

I typically write my lines like this "Copyright (c) 2017, 2020 Ma_Sys.ma. For further info send an e-mail to Ma_Sys.ma@web.de."

Here is the `copyright` file from the `bazel-bootstrap` Debian package: <https://metadata.ftp-master.debian.org/changelogs//main/b/bazel-bootstrap/bazel-bootstrap_3.5.1+ds-3_copyright>, to pick a random of the included `Copyright` fields:

~~~
Copyright: 1998-2010 AOL Inc.
           2006-2009 James Murty
           2010-2018 Amazon.com, Inc. or its affiliates
           2011-2018 Amazon Technologies, Inc.
~~~

Basically I am looking for the correct value to fill into the `Copyright` field for Alire.

**Comment by mosteo** [comment, marked as answer]
Hi again @m7a . You can use:

2017-2022 Alejandro R. Mosteo

If it's the same to you, we're nearing a release with important changes, if you want to start packaging with that one instead.

Thanks for the interest in the project.

**Reply by m7a** [reply]
Thank you very much for the information. My package is most liekly to be independent of any distribution's -- I use a simplified packaging scheme where I describe the entire package in a single XML file; I can add the link here as soon as its ready if wanted. 

I will watch out for the new release but it should not be hard to upgrade the package later, as long as the `gprbuild`-based compilation remains roughly the same.

**Comment by mosteo** [comment]
Yes, we intend to keep the build simple around `gprbuild` for as long as possible.
